src/app/checkout/step-2/bank-account/bank-account.tpl.html

```diff
@@ -82,7 +82,7 @@ <h4 class="panel-title  border-bottom-small  visible" translate>Bank Account Pay
             <div class="panel-body">
               <div class="row">
                 <div class="col-sm-8">
-                  <img src="/assets/img/check-sample.png" alt="Credit Card Icon"/>
+                  <img src="/assets/images/check-sample.png" alt="Credit Card Icon"/>
```

Comment:
The variable will just be the s3 urls right? Do you think adding interaction between AEM and angular is better? I was thinking just add hardcoded strings for the buckets...